scripts/OGC_tests/configure_TE_BASE.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Preparing TE_BASE"
+mkdir TE_BASE
+
+export TE_BASE=/home/travis/build/mozilla-sensorweb/sensorthings/TE_BASE
```

Comment:
Would this be a valid solution? Using:  
COMMON_PATH=$PWD
TE_BASE=$COMMON_PATH/TE_BASE

and same for the rest of the paths used.